bcreg-aca/config/README.md

```diff
@@ -186,6 +195,13 @@ issuers:
     - description: Permit
       schema: my-permit.my-organization.ca
       issuer_url: http://localhost:5001/my-organization/my-permit
+
+      details:
+        url_en: $APPLICATION_URL_INCORP
+        highlighted_attributes:
+          - reason_description
+        credential_title: Registration
```

Comment:
make sure the credential_title references an attribute key such as `reason_description`. Usually registration credentials don't have a credential title